config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -252,25 +255,26 @@ spec:
       priorityClassName: dynatrace-high-priority
       volumes:
       # This volume is where the registrar registers the plugin with kubelet
-      - hostPath:
-          path: /var/lib/kubelet/plugins_registry/
+      - name: registration-dir
+        hostPath:
+          path: {{ include "dynatrace-operator.CSIRegistrationDir" . }}
           type: Directory
-        name: registration-dir
         # This volume is where the socket for kubelet->driver communication is done
       - name: plugin-dir
         hostPath:
-          path: /var/lib/kubelet/plugins/csi.oneagent.dynatrace.com
+          path: {{ include "dynatrace-operator.CSIPluginDir" . }}
           type: DirectoryOrCreate
         # This volume is where the driver mounts volumes
-      - hostPath:
-          path: /var/lib/kubelet/pods
+      - name: mountpoint-dir
+        hostPath:
+          path: {{ include "dynatrace-operator.CSIMountPointDir" . }}
           type: DirectoryOrCreate
-        name: mountpoint-dir
         # This volume is where the driver persists data on the node
-      - hostPath:
-          path: /var/lib/kubelet/plugins/csi.oneagent.dynatrace.com/data
+      - name: dynatrace-oneagent-data-dir
```

Comment:
we could combine this volume with the `plugin-dir` volume and mount the subfolder via `mountPath`